source/common/config/cds_json.cc

```diff
@@ -140,6 +140,9 @@ void CdsJson::translateCluster(const Json::Object& json_cluster,
     cluster.set_type(envoy::api::v2::Cluster::ORIGINAL_DST);
   } else {
     ASSERT(string_type == "sds");
+    if(!eds_config.has_value()){
+      throw EnvoyException("fetching invalid Optional value");
```

Comment:
In the description you mention an new exception in `retry_state_impl.cc`; I don't see that, but I see this.. is this what you meant?

Reply:
Yea, updated the description. Sorry about that.